copyright

```diff
@@ -541,6 +541,13 @@ Copyright: Becca Tommaso (tommasobecca03@gmail.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser and Evan Fluharty (under the same license).
 
+Files:
+ images/hardpoint/annihilator?turret*
+ images/hardpoint/ravager?turret*
+Copyright: Becca Tommaso (tommasobecca03@gmail.com)
+License: CC-BY-SA-4.0
+Comment: Derived from works by Michael Zahniser (under the same license).
```

Comment:
Add these to the files at line 333 instead of having another entry with all the same information.

Reply:
Done.